README.md

```diff
@@ -285,47 +306,42 @@ SDC is composed of several pre-built components:
   a slightly customized build of vanilla SmartOS for SDC.
 - *Virtual machine images* for SDC services (e.g. imgapi, vmapi, adminui), which
   are provisioned as VMs at install time.
-- Agents, which are bundled into a [single
+- Agents bundled into a [single
   package](https://github.com/joyent/sdc-agents-installer)
-  that can then be installed into the global zone of Compute Nodes.
+  installed into the global zone of each compute node.
 
 Each component is built separately and then all are combined into CoaL and USB
 builds (see the preceding sections) via the [sdc-headnode
-repository](https://github.com/joyent/sdc-headnode). Built components are typically
-stored in a [Manta object store](https://github.com/joyent/manta), e.g.
-[Joyent's public Manta](https://www.joyent.com/products/manta), and pulled from
-there. For example, Joyent's core builds push to
+repository](https://github.com/joyent/sdc-headnode). The built components are
+typically stored in a [Manta object store](https://github.com/joyent/manta),
+e.g. [Joyent's public Manta](https://www.joyent.com/products/manta),
+and pulled from there. For example, Joyent's builds push to
 `/Joyent_Dev/public/builds` in Joyent's public Manta in us-east-1
 (<https://us-east.manta.joyent.com/>).
 
-You can build your own CoaL and USB on Mac or SmartOS (see the [sdc-headnode
-README](https://github.com/joyent/sdc-headnode#readme)). However, all other
-SDC components must be built using a running SDC (e.g. on the [Joyent Cloud](https://www.joyent.com/products/compute-service)
-or in a local CoaL). See [the building
-document](./docs/developer-guide/building.md) for details on building each of
-the SDC components.
+You can build your own CoaL and USB images on Mac or SmartOS (see the
+[sdc-headnode README](https://github.com/joyent/sdc-headnode#readme)). However,
+all other SDC components must be built using a running SDC
+(e.g. on the [Joyent Cloud](https://www.joyent.com/products/compute-service)
+or in a local CoaL). See [the building document](./docs/developer-guide/building.md)
+for details on building each of the SDC components.
 
 
 ## Contributing
 
-To report bugs or request features, submit issues to the [joyent/sdc
-project](https://github.com/joyent/sdc/issues). If you're contributing code,
-make a pull request to the appropriate repo (see [the repo
-overview](./docs/developer-guide/repos.md)). If you're contributing something
-substantial, you should first contact developers on the
-[sdc-discuss mailing list](mailto:sdc-discuss@lists.smartdatacenter.org)
+To report bugs or request features, submit issues here on
```

Comment:
Would like to go with same wording as I suggest in https://github.com/joyent/sdc/pull/48#discussion_r25988956 .